### ISSUE TITLE
[BUG] Game Mode: Could not detect video card...

### ISSUE THREAD

**Issue by Mumbis** [nicehash/NiceHashQuickMiner#680]
**Describe the bug**
Sometimes after a day of mining I try to activate gaming mode and get error message "Could not detect video card with connected display or you may be using older video card for display!"

**To Reproduce**
1. Activate Game Mode after one night of mining

**Expected behavior**
Game mode should be activated. Most of the time it works.

**Version affected (please complete the following information):**
ver 0.5.2.0 RC

**NVIDIA driver version**
496.49

**Hardware**
Nvidia RTX 2060 Super in a normal gaming PC.
AMD 3700X CPU

[exported_logs.zip](https://github.com/nicehash/NiceHashQuickMiner/files/7492032/exported_logs.zip)
![NH](https://user-images.githubusercontent.com/29867367/140640141-d70d45d3-a9ba-4687-b6a6-04947c77ea08.png)

**Comment by PabloDons** [comment]
Also Experiencing this

**Comment by RocketPop** [comment]
Still happening for me (now much more frequently) on Nvidia drivers 497.29 w/an Intel cpu (windows 10)

**Comment by ialexryan** [comment]
I saw this error for the first time a week or two ago, and I now see it almost every time I try to enter game mode. I think it might have started after an Nvidia driver update but I'm not sure. Only fix I've found so far is rebooting my whole computer.

Running NiceHash v0.5.2.0 Stable and Nvidia v497.09 with a 3070 and an AMD CPU on Windows 10.
![image](https://user-images.githubusercontent.com/2292782/147794101-818ef788-8907-46ef-aa11-f504cc140ed1.png)
